### DISCUSSION TITLE
v2.0.7: Merge pull request #148 from libsgh/dev

### DISCUSSION THREAD

**Discussion by libsgh** [libsgh/PanIndex#149] [General]
**为方便以后配置扩展，此版本主要是对后台配置进行了重写，另外只有mdui主题做了功能增强，请尽量使用该主题**

##### Bug Fixes

- 后台配置优化：自定义账号、文件列表排序、防盗链、隐藏&密码文件
- 修复『天翼云盘』上传接口、取消文件夹下载
- 解决列表页图片预览加载优化 
- 优化密码输入、主题切换记住功能 
- 去除无用的定时任务
- 预览文件后缀可配置
- 『本地模式』打包下载临时文件清理

##### Features
- 添加分享短链&二维码
- 添加下载链接一键提取
- 『阿里云盘』视频支持转码播放
- pdf预览支持手机端
- 『阿里云盘』支持文件夹下载
- 支持epub格式的电子书预览
- 添加分享短链&二维码
- 添加下载链接一键提取
- 『阿里云盘』视频支持转码播放
- 列表页支持`HEAD.md`渲染

<hr /><em>This discussion was created from the release <a href='https://github.com/libsgh/PanIndex/releases/tag/v2.0.7'>v2.0.7: Merge pull request #148 from libsgh/dev</a>.</em>

**Comment by ivkeji** [comment]
笑出强大，没有蛀牙。作者666
